docker/docker-compose.yaml

```
@@ -30,7 +30,7 @@ services:
 
   test:
     <<: *common
-    command: /bin/bash -xcl "swift $${SWIFT_TEST_VERB-test} $${WARN_AS_ERROR_ARG-} $${SANITIZER_ARG-} $${IMPORT_CHECK_ARG-}"
+    command: /bin/bash -xcl "swift $${SWIFT_TEST_VERB-test} $${WARN_AS_ERROR_ARG-} $${SANITIZER_ARG-} $${IMPORT_CHECK_ARG-} -Xswiftc -strict-concurrency=complete"
```

To follow the convention here we should probably replace this with...
```suggestion
    command: /bin/bash -xcl "swift $${SWIFT_TEST_VERB-test} $${WARN_AS_ERROR_ARG-} $${SANITIZER_ARG-} $${IMPORT_CHECK_ARG-} $${STRICT_CONCURRENCY_ARG-}"
```

...and then specify it in:
- `docker/docker-compose.2204.58.yaml`
- `docker/docker-compose.2204.59.yaml`

If you look in those files, that's how `-Xswiftc -warnings-as-errors` is passed.

---

I did it for `docker-compose.2204.main.yaml` too as i couldn't think of a reason why that file should be left out. Let me know if it shouldn't be there.